hatch_build.py

```diff
@@ -148,24 +148,18 @@
 
 DOC_EXTRAS: dict[str, list[str]] = {
     "doc": [
-        "astroid>=2.12.3,<3.0",
+        "astroid>=3",
         "checksumdir>=1.2.0",
         "click>=8.1.8",
-        # Docutils 0.17.0 converts generated <div class="section"> into <section> and breaks our doc formatting
-        # By adding a lot of whitespace separation. This limit can be lifted when we update our doc to handle
-        # <section> tags for sections
-        "docutils<0.17,>=0.16",
+        "docutils>=0.21",
         "sphinx-airflow-theme>=0.0.12",
```

Comment:
Before merging, we should create a new version for `sphinx-airflow-theme` (0.1.0) after merging https://github.com/apache/airflow-site/pull/1103

Reply:
As it is now there:
```suggestion
        "sphinx-airflow-theme>=0.1.0",
```

Reply:
Cool :)